src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-sys-dist.mdx

```diff
@@ -0,0 +1,50 @@
+  - Amazon integrations
+  - AWS integrations list
+metaDescription: "New Relic's AWS System Manager Distributor monitoring integration: what data it reports, and how to enable it."
+---
+
+[New Relic infrastructure integrations](/docs/infrastructure/integrations-getting-started/getting-started/introduction-infrastructure-integrations) include an integration for reporting your application and AWS System Manager Distributor data to New Relic observability platform. This document explains the integration's features, how to activate it, and what data can be reported.
+
+## Features
+Collect and send telemetry data to New Relic from your apps and services from AWS System Manager Distributor. Monitor your services, query incoming data, and build dashboards to observe everything at a glance. 
+
+## Activate integration
+
+To enabl this integration:
+
+1. Place your [New Relic license key](/docs/apis/intro-apis/new-relic-api-keys/) in the AWS Systems Manager Parameter Store. Specify `/newrelic-infra/license` as `Name` in the parameter field and choose the `SecureString` option in `Type`. Place your key in `Value` field.
+2. From the [IAMV console](https://console.aws.amazon.com/iam/home?#/roles) create a role utilizing `AmazonSSMManagedInstanceCore` policy.
+3. If the AMI being used for the instance(s) does not have AWS CLI agent preinstalled, install it manually:
+  ```
+  curl "https://awscli.amazonaws.com/awscli-exe-linux-{x86_64|aarch64}.zip" -o "awscliv2.zip"
+  unzip awscliv2.zip
+  sudo ./aws/install
+  ```
+4. If the AMI does not have SSM agent preinstalled, install it manually or via a template:
+  ```
+  sudo yum install -y https://s3.<var>{region}</var>.amazonaws.com/amazon-ssm-<var>{region}</var>/latest/linux_amd64/amazon-ssm-agent.rpm
+  ```
+5. On the Systems Manager interface select **Third party** and select the **NewRelicInfraAgent** tile.
+6. Choose your preffered installation, then select **Run a command** and in the **Targets** pane click **Choose instances manually**. Select all the instances you want to monitor and then click **Run**.
+
+1. Generate some traffic for your app, then wait a few minutes for your app to send data to New Relic.
+2. Explore your app's data in the [our UI](one.newrelic.com).
+
+## Configuration and polling 
+
+You can change the polling frequency and filter data using [configuration options](/docs/integrations/new-relic-integrations/getting-started/configure-polling-frequency-data-collection-cloud-integrations).
+
+[Default polling information](/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-polling-intervals-infrastructure-integrations) for the AWS integrations:
+- New Relic polling interval: 5 minutes
+- Amazon CloudWatch data interval: 1 minute or 5 minutes
+
+
+## Find and use data
+To find your integration data, go to your account on [one.newrelic.com ](one.newrelic.com).
+
+For metrics:
+On the left column select **Data explorer > Metrics** and filter by `aws.???????`
```

Comment:
@4demos are there metrics we can filter for this integration?

Reply:
I'd refer here to our Infrastructure Agent page. SMD is just a way to install our Infrastructure Agent. After it's installed the rest is the same regardless the way it has been installed